.github/actions/get-latest-pr-number/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Get Latest Merged PR Number
+description: Get the latest merged PR number from the release branch, this is the production candidate
+branding:
+  icon: git-pull-request
+  color: blue
+
+inputs:
+  token:
+    description: Specify token (GH or PAT), instead of inheriting one from the calling workflow
+    default: ${{ github.token }}
+
+outputs:
+  pr:
+    description: "Latest merged pull request number"
+    value: ${{ steps.vars.outputs.pr }}
+
+runs:
+  using: composite
+  steps:
+    - id: vars
+      shell: bash
+      run: |
+        git fetch origin
+
+        release_branch="${{ github.event.pull_request.head.ref }}"
+        echo "Detected release branch: $release_branch"
+
+        latest_pr=$(git log origin/$release_branch --merges --pretty=format:'%s' | grep -oP '(?<=#)\d+' | head -n 1)
```

Comment:
Running this locally and looking through our git log it appears we have fast-forward merges so this returns empty. Removing the `--merges` makes this return, what I think is the correct list, and that does evaluate to the latest PR number. 

At the time of testing, release/noble-sea-lemon had a merge PR #724, and that was returned (`724`) after removing the `--merges` flag

Reply:
Nice find!  I'll commit that change.